static/sass/_patterns_inline-list--compact.scss

```diff
@@ -1,6 +1,8 @@
 @mixin p-inline-list--compact {
   .p-inline-list--compact {
+    // sass-lint:disable placeholder-in-extend
```

Comment:
Shouldn't `// sass-lint:disable-line ...` work just fine for one-liners like this?

Reply:
Good catch, will fix